processor/processor.go

```diff
@@ -0,0 +1,13 @@
+package processor
+
+import (
+	"github.com/algorand/go-algorand/ledger/ledgercore"
+	"github.com/algorand/go-algorand/rpcs"
+)
+
+// Processor is the block processor interface
+type Processor interface {
+	Process(cert *rpcs.EncodedBlockCert) error
+	SetHandler(handler func(block *ledgercore.ValidatedBlock) error)
```

Comment:
Do we still need this function?

Reply:
I think in practice it will never be needed. But you can add it back when it is!